torch/fx/experimental/symbolic_shapes.py

```diff
@@ -144,16 +110,15 @@ def create_contiguous(shape):
     return list(reversed(strides))
 
 def _handle_sym_dispatch(func, args, kwargs):
```

Comment:
Any reason not to put this in `_sym_dispatch_mode.py`?

Reply:
I wanted to make changes that self-contained as possible. Of course this was before I had to move 1k lines into its own file. I'll move this one now.